transforms/react-to-react-dom.js

```diff
@@ -285,24 +316,29 @@ module.exports = function(file, api) {
     coreUses += root.find(j.JSXElement).size();
 
     function insertRequire(name, path) {
-      var req = j.callExpression(
-        j.identifier('require'),
-        [j.literal(path)]
-      );
-      requireAssignments.forEach(function(requireAssignment) {
-        requireAssignment.parent.insertAfter(
-          j.expressionStatement(
-            j.assignmentExpression('=', j.identifier(name), req)
-          )
+      if (coreRequireDeclarator) {
+        var req = j.callExpression(
+          j.identifier('require'),
+          [j.literal(path)]
         );
-      });
-      coreRequireDeclarator.parent.insertAfter(j.variableDeclaration(
-        coreRequireDeclarator.parent.value.kind,
-        [j.variableDeclarator(
-          j.identifier(name),
-          coreRequireDeclarator.value.init ? req : null
-        )]
-      ));
+        requireAssignments.forEach(function(requireAssignment) {
+          requireAssignment.parent.insertAfter(
+            j.expressionStatement(
+              j.assignmentExpression('=', j.identifier(name), req)
+            )
+          );
+        });
+        coreRequireDeclarator.parent.insertAfter(j.variableDeclaration(
+          coreRequireDeclarator.parent.value.kind,
+          [j.variableDeclarator(
+            j.identifier(name),
+            coreRequireDeclarator.value.init ? req : null
+          )]
+        ));
+      } else {
+        var imp = j.importDeclaration([j.importDefaultSpecifier(j.identifier(name))], j.literal(path))
```

Comment:
this should share where we want to emit
```js
import ReactDOM, { render } from 'react-dom';
```
instead of
```js
import ReactDOM from 'react-dom';
import { render } from 'react-dom';
```